libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/model/PrimaryManeuver.kt

```diff
@@ -37,6 +38,7 @@ import com.mapbox.api.directions.v5.models.BannerInstructions
  */
 
 class PrimaryManeuver private constructor(
+    val id: String,
```

Comment:
See comment above re: not sure if I completely understand why this is needed.

Reply:
Same as above